instrumentation/camunda/camunda-7.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/camunda/v7_0/behavior/CamundaBehaviorSingletons.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.camunda.v7_0.behavior;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.InstrumenterBuilder;
+import io.opentelemetry.instrumentation.camunda.v7_0.behavior.CamundaBehaviorSpanNameExtractor;
+import io.opentelemetry.instrumentation.camunda.v7_0.common.CamundaCommonRequest;
+import io.opentelemetry.instrumentation.camunda.v7_0.common.CamundaVariableAttributeExtractor;
+
+public class CamundaBehaviorSingletons {
+
+  private static final Instrumenter<CamundaCommonRequest, String> instrumenter;
+
+  private static final OpenTelemetry opentelemetry;
+
+  static {
+    opentelemetry = GlobalOpenTelemetry.get();
+
+    InstrumenterBuilder<CamundaCommonRequest, String> builder =
```

Comment:
Are you actually using this String return? most usages I see you just passing "NA" in, so curious if it's actually needed or if it coud be void
```suggestion
    InstrumenterBuilder<CamundaCommonRequest, Void> builder =
```